docs/userGuide/components/panel.md

```diff
@@ -0,0 +1,225 @@
+
+```html
+<panel header="Header" alt="Alternative Header" minimized>
+  ...
+</panel>
+```
+</tip-box>
+<br>
+
+#### With `expanded` attribute, panel could be expanded by default.
+
+<tip-box border-left-color="#00B0F0">
+  <i style="font-style: normal; font-weight: bold; color: dimgray">Example</i><br>
+  <panel header="Header" alt="Alternative Header" expanded>
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias dolores excepturi expedita illum impedit ipsa labore qui veniam. Blanditiis exercitationem id ipsum libero molestiae, necessitatibus unde? Amet fugiat fugit molestias?
+  </panel>
+</tip-box>
+
+<tip-box border-left-color="black">
+<i style="font-style: normal; font-weight: bold; color: dimgray">Markup</i>
+
+```html
+<panel header="Header" alt="Alternative Header" expanded>
+  ...
+</panel>
+```
+</tip-box>
+<br>
+
+#### Panel has many types that change its appearance.
+
+<tip-box border-left-color="#00B0F0">
+  <i style="font-style: normal; font-weight: bold; color: dimgray">Example</i><br>
+  <code>primary</code> type<br>
+  <panel header="#### primary - Click to expand" type="primary">
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias dolores excepturi expedita illum impedit ipsa labore qui veniam. Blanditiis exercitationem id ipsum libero molestiae, necessitatibus unde? Amet fugiat fugit molestias?
+  </panel>
+  <br>
+  <code>seamless</code> type<br>
+  <panel header="#### seamless - Click to expand" type="seamless">
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias dolores excepturi expedita illum impedit ipsa labore qui veniam. Blanditiis exercitationem id ipsum libero molestiae, necessitatibus unde? Amet fugiat fugit molestias?
+  </panel>
+</tip-box>
+
+<tip-box border-left-color="black">
+<i style="font-style: normal; font-weight: bold; color: dimgray">Markup</i>
+
+```html
+<panel header="#### primary - Click to expand" type="primary">
+  ...
+</panel>
+<panel header="#### seamless - Click to expand" type="seamless">
+  ...
+</panel>
+```
+</tip-box>
+<br>
+
+#### Show/Hide buttons using `no-switch` or `no-close`
+
+<tip-box border-left-color="#00B0F0">
+  <i style="font-style: normal; font-weight: bold; color: dimgray">Example</i><br>
+  <panel header="Header" no-switch>
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias dolores excepturi expedita illum impedit ipsa labore qui veniam. Blanditiis exercitationem id ipsum libero molestiae, necessitatibus unde? Amet fugiat fugit molestias?
+  </panel>
+  <panel header="Header" no-close>
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias dolores excepturi expedita illum impedit ipsa labore qui veniam. Blanditiis exercitationem id ipsum libero molestiae, necessitatibus unde? Amet fugiat fugit molestias?
+  </panel>
+</tip-box>
+
+<tip-box border-left-color="black">
+<i style="font-style: normal; font-weight: bold; color: dimgray">Markup</i>
+
+```html
+<panel header="Header" no-switch>
```

Comment:
Perhaps the header can be "This panel has no switch!", rather than just a generic "Header".